randovania/game_description/pickup/standard_pickup.py

```diff
@@ -20,26 +20,86 @@
 @dataclass(frozen=True)
 class StandardPickupDefinition(JsonDataclass, DataclassPostInitTypeCheck):
     game: RandovaniaGame = dataclasses.field(metadata={"init_from_extra": True})
+    """The game this pickup comes from."""
+
     name: str = dataclasses.field(metadata={"init_from_extra": True})
+    """The name of the pickup."""
+
     pickup_category: PickupCategory = dataclasses.field(metadata={"init_from_extra": True})
+    """The precise category for this pickup."""
+
     broad_category: PickupCategory = dataclasses.field(metadata={"init_from_extra": True})
+    """The broad category for this pickup."""
+
     model_name: str
+    """The name of the model that should be used by default for this pickup."""
+
     offworld_models: frozendict[RandovaniaGame, str]
+    """A dictionary defining the name of the model for other games if this pickup is in their world."""
+
     progression: tuple[str, ...]
+    """
+    Defines item resources (as short names) that collecting this pickup provides.
+    If this tuple contains only one resource, then every collection will give that resource.
+    If it contains more than one, then every time the pickup will be collected, it will give the Nth resource.
+    """
+
     preferred_location_category: LocationCategory
+    """The category for the preferred location."""
```

Comment:
I hate this kind of docstring, it adds nothing but a string that can become outdated.

Please explain what that is.

Reply:
Shouldn't the LocationCategory class itself explain on what that is?
Not opposed to it, it's just the way I usually have seen things documented.

Reply:
`LocationCategory` determines what kind of location a pickup node is. Don't see why it should explain why a standard pickup has a preference for some kind of category to me